### PR TITLE
Initially add all .settings

### LINKED ISSUE #594: All .settings folders are listed by git as "untracked"
...they should be added to .gitignore

### PR DESCRIPTION
**Pull request by Flugtiger**
Fixes #594

### CODE REVIEW

**Comment by ruspl-afed** [comment]
@wimjongman do we have an upper bound for a PR volume (>1000 LOC or similar)?

**Comment by Flugtiger** [comment]
Just to clearify: I did not touch any of those files. They were all created by m2e automatically. 😉

**Comment by ruspl-afed** [comment]
> Just to clearify: I did not touch any of those files. They were all created by m2e automatically. 😉

yes, it is clear. I'm just using this opportunity to clarify the rules :)

**Comment by Flugtiger** [comment]
Maybe we should discuss rules for PRs on the mailing list. I have some thoughts on it as well.

**Comment by wimjongman** [comment]
> 
> 
> @wimjongman do we have an upper bound for a PR volume (>1000 LOC or similar)?
Committers do not have an upper bound.

File: UI/org.eclipse.birt.report.designer.ui.editors/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
+org.eclipse.jdt.core.compiler.compliance=1.6
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.release=disabled
+org.eclipse.jdt.core.compiler.source=1.6
```
Comment (wimjongman) [inline UI/org.eclipse.birt.report.designer.ui.editors/.settings/org.eclipse.jdt.core.prefs:8]: I see a lot of source and target 1.6 This needs to be discussed as well.

**Pull request merged**